plugin/evm/validators_manager_test.go

```diff
@@ -0,0 +1,65 @@
+package evm
+
+import (
+	"encoding/binary"
+	"math/rand"
+	"testing"
+
+	"github.com/flare-foundation/flare/ids"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
```

Comment:
```suggestion
import (
	"encoding/binary"
	"math/rand"
	"testing"

	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"

	"github.com/flare-foundation/flare/ids"
)
```

Reply:
This import list has been removed, because the function from this file has been moved to vm_test as per my comment https://github.com/flare-foundation/coreth/pull/21#discussion_r830626604